### PR TITLE
Add feature to initialize with swl from swlheight

### PR DESCRIPTION
**Pull request by berland**
* Refactor simple_J, extract to utils

### CODE REVIEW

File: tests/test_utils_capillarypressure.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+import numpy as np
+import pytest
+from hypothesis import given
+import hypothesis.strategies as st
+
+from pyscal.constants import EPSILON
+from pyscal.utils import capillarypressure
+
+# pyscal.utils.relperm.crosspoint() is tested by test_wateroil and test_gasoil.
```
Comment (berland) [inline tests/test_utils_capillarypressure.py:12]: rm this liine

File: pyscal/utils/capillarypressure.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+
+def _height_to_pc(height, drho, g):
+    """From height above free water level, multiplication
+    with density difference and gravity gives capillary pressure.
+
+    Args:
+        height (float): Height above free water level in meters.
+        drho (float): density difference in g/cc
+        g (float): gravitational acceleration, in m/s²100
```
Comment (berland) [inline pyscal/utils/capillarypressure.py:61]: stray number 100

File: pyscal/factory.py [hunk 3]
```diff
@@ -154,6 +156,22 @@ def create_water_oil(params=None):
         params = filter_nan_from_dict(params)
 
         usedparams = set()
+
+        # Check if we should initialize swl from a swlheight parameter:
```
Comment (berland) [inline pyscal/factory.py:160]: Add this for GasOil/WaterOilGas as well, since it uses swl

**Review by rnyb — APPROVED** [review]
Looks ok  based on quick scan

**Pull request merged**